hello_wifi/mix.exs

```diff
@@ -70,14 +75,11 @@ defmodule HelloWifi.Mixfile do
      "deps.loadpaths":  ["deps.loadpaths", "nerves.loadpaths"]]
   end
 
+  def kernel_modules("rpi0"), do: ["brcmfmac"]
```

Comment:
I think that the systems in the raspberry pi series should behave as similarly as possible. I'll look at including the brcmfmac driver as a module instead for the rpi0 system.